dbc2val/dbcfeeder.py

```diff
@@ -239,39 +253,31 @@ def _run(self):
                         log.error("Failed to register datapoints", exc_info=True)
                         continue
             try:
-                can_signal, can_value = self._can_queue.get(timeout=1)
-                for target in self._mapper[can_signal]["targets"]:
-                    value = self._mapper.transform(can_signal, target, can_value)
-                    if value != can_value:
-                        log.debug(
-                            "  transform({}, {}, {}) -> {}".format(
-                                can_signal, target, can_value, value
-                            )
-                        )
-                    # None indicates the transform decided to not set the value
-                    if value is None:
-                        log.warning(
-                            "failed to transform({}, {}, {})".format(
-                                can_signal, target, can_value
-                            )
-                        )
-                    else:
-                        # get values out of the canreplay and map to desired signals
-                        log.debug("Updating DataPoint(%s, %s)", target, value)
-                        if self._server_type is ServerType.KUKSA_DATABROKER:
+                vss_observation = self._can_queue.get(timeout=1)
+                vss_mapping = self._mapper.get_vss_mapping(vss_observation.dbc_name, vss_observation.vss_name)
+                value = vss_mapping.transform_value(vss_observation.raw_value)
+                if value is None:
+                    log.warning(f"Value ignored for  dbc {vss_observation.dbc_name} to VSS {vss_observation.vss_name},"
+                                f" from raw value {value} of type {type(value)}")
+                elif not vss_mapping.change_condition_fulfilled(value):
+                    log.info(f"Value condition not fulfilled for VSS {vss_observation.vss_name}, value {value}")
```

Comment:
have not tested it yet, but might the log level be too high? In the past we observed if just printing some message xxx times per second will slow down the feeder a lot/generate a lot of load. And just from looking at the code, this is not an error condition, but something that we might expect to encounter regularly. So maybe verbose?.

Reply:
Could be reduced to debug.

For now if running on info-level you get typically one line every time a signal is sent to broker (and if here is value is discarded), which could be quite many. So the question is - how verbose do we want to be on info-level, do we expect info-level to be usable for "real deployments" where performance is important?

Reply:
As of today default is INFO for dbcfeeder. I have no problems using DEBUG on recurring messages. But then , if you would be interested in knowing if any signals (at all) has been sent to databroker then you would need to change to DEBUG, and then you would get a lot of other messages as well.